src/query/pipeline/sources/Cargo.toml

```diff
@@ -35,3 +35,4 @@ parking_lot = "0.12.1"
 serde_json = { workspace = true }
 similar-asserts = "1.4.2"
 tracing = "0.1.36"
+xml-rs = "0.8.4"
```

Comment:
Add xml-rs in `[package.metadata.cargo-machete]` to make `cargo-machete` happy.

Like

https://github.com/datafuselabs/databend/blob/1dd316bd4d22c1b38ab2cec04f5a23a7e8f20bd5/src/query/config/Cargo.toml#L16-L17